src/libp2p_crypto.erl

```diff
@@ -11,27 +11,45 @@
 %% the type of keythat follows in the binary (KEYTYPE).
 -define(KEYTYPE_ECC_COMPACT, 0).
 -define(KEYTYPE_ED25519, 1).
+-define(KEYTYPE_MULTISIG, 2).
 -define(NETTYPE_MAIN, 0).
 -define(NETTYPE_TEST, 1).
 
+-define(MULTISIG_SIG_LEN_BYTES,  8). % TODO Will we ever have a sig len > 65535?
```

Comment:
```suggestion
-define(MULTISIG_SIG_LEN_BYTES,  8). % TODO Will we ever have a sig len > 256?
```

We should check how big BLS signatures are, but presumably we could always interpret the sig length in a key specific way if we needed to?

Reply:
Good catch - was still based on bits.

As for key-specific way: do you mean to have a static list of expected sig lengths per key type?

Reply:
Well, the meaning of the length field doesn't have to be in bytes, it can be in some keytype specific unit (eg uint64s)